resources/definitions/hellbot_magna_I.def.json

```
@@ -0,0 +1,33 @@
+ {
+ 	"version": 2,
+ 	"name": "Hellbot Magna I",
+ 	"inherits": "fdmprinter",
+ 	"metadata": {
+ 		"visible": true,
+ 		"author": "MUX team",
+ 		"manufacturer": "Hellbot",
+ 		"file_formats": "text/x-gcode",
+ 		"platform": "hellbot_magna.obj",
+ 		"platform_texture": "hellbot.png",
+ 		"platform_offset": [0, -1, 0],
+ 		"has_materials": true,
+ 		"machine_extruder_trains": {
+ 			"0": "hellbot_magna_i_extruder"
+ 		}
+ 	},
+ 	"overrides": {
+ 		"machine_name": {
+ 			"default_value": "Hellbot Magna I"
+ 		},
+ 		"machine_width": {
+ 			"default_value": 230
```

The specifications indicates that the X dimension is 220mm: https://hellbot.xyz/magna-1/
```suggestion
 			"default_value": 220
```